src/System.Text.RegularExpressions/tests/RegexGroupCollectionTests.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System.Collections.Generic;
+using System.Text.RegularExpressions;
+using Xunit;
+
+public class RegexGroupCollectionTests
+{
+    [Fact]
+    public static void GetGroupTest()
+    {
+        var caps = new Dictionary<int, int>
+        {
+            { 0, 0 }
+        };
+        var groups = new GroupCollection(null, caps);
```

Comment:
It looks like because this PR was created before the Jenkins was enabled, it doesn't see it.  If you close and reopen the PR (this shouldn't kill any history or info) I think it will pick it up.